generation/WinSDK/enums.json

```diff
@@ -29813,5 +29813,50 @@
         "field": "dwFlags"
       }
     ]
+  },
+  {
+    "name": "DROPEFFECT",
+    "flags": false,
+    "type": "uint",
+    "members": [
+      {
+        "name": "DROPEFFECT_NONE",
+        "value": 0
```

Comment:
Why bother including `value` here? In my own PR that addresses the issue I filed (#940), I was able to omit them.